crates/nu-engine/src/filesystem/path.rs

```diff
@@ -1,6 +1,25 @@
 use std::io;
 use std::path::{Component, Path, PathBuf};
 
+pub fn resolve_dots<P>(path: P) -> PathBuf
```

Comment:
Feels like this is something we'd want to use for all our paths that we show to the user

Reply:
(or something like it)